A.css

```diff
@@ -0,0 +1,4 @@
+/* stylelint-disable indentation */
```

Comment:
- [ ] This is bad practice of passing linter, you can correct it via running `npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x` then `npx stylelint "**/*.{css,scss}"`, you can correct this linter error manually also.